homeassistant/components/eq3btsmart/const.py

```diff
@@ -0,0 +1,6 @@
+""" Constants for EQ3 Bluetooth Smart Radiator Valves """
+
+PRESET_PERMANENT_HOLD = "PermanentHold"
+PRESET_NO_HOLD = "NoHold"
+PRESET_OPEN = "Open"
+PRESET_CLOSED = "Closed"
```

Comment:
```suggestion
PRESET_PERMANENT_HOLD = "permanent_hold"
PRESET_NO_HOLD = "no_hold"
PRESET_OPEN = "open"
PRESET_CLOSED = "closed"
```

Reply:
Used naming convention used in [https://github.com/home-assistant/core/blob/a59460a23336627d0bc12b1eefffdaa516e55e87/homeassistant/components/lyric/const.py](url)

But I see that e.g [https://github.com/home-assistant/core/blob/a59460a23336627d0bc12b1eefffdaa516e55e87/homeassistant/components/nuheat/climate.py](url) uses different naming convention.

So... What to use?

Reply:
Well, the reason I made this review comment, is that this matches the HA `PRESET_BOOST` and `PRESET_AWAY` that follow the lower case snake style convention. Right now, this PR is creating a mixed bag.

Reply:
good point :-)